build_tools/github_actions/test_executable_scripts/test_miopen.py

```diff
@@ -183,12 +183,13 @@
 # If smoke tests are enabled, we run smoke tests only.
 # Otherwise, we run the normal test suite
 test_type = os.getenv("TEST_TYPE", "full")
-if test_type == "smoke":
-    test_filter = "--gtest_filter=" + ":".join(smoke_filter)
-else:
-    test_filter = (
-        "--gtest_filter=" + ":".join(positive_filter) + "-" + ":".join(negative_filter)
-    )
+if test_type != "full":
+    if test_type == "smoke":
+        test_filter = "--gtest_filter=" + ":".join(smoke_filter)
+    else:
+        test_filter = (
+            "--gtest_filter=" + ":".join(positive_filter) + "-" + ":".join(negative_filter)
+        )
```

Comment:
noticing errors such as:

```
Traceback (most recent call last):
  File "/__w/TheRock/TheRock/build_tools/github_actions/test_executable_scripts/test_miopen.py", line 195, in <module>
    cmd = [f"{THEROCK_BIN_DIR}/miopen_gtest", test_filter]
                                              ^^^^^^^^^^^
NameError: name 'test_filter' is not defined
```

should this be:

```suggestion
test_filter = ""
if test_type == "smoke":
        test_filter = "--gtest_filter=" + ":".join(smoke_filter)
```